client_test.go

```diff
@@ -1283,3 +1283,30 @@ func TestGocloak_GetUsersByRoleName(t *testing.T) {
 		userID,
 		(*users)[0].ID)
 }
+
+// -----------
+// Events
+// -----------
+
+func GetEvents(t *testing.T, client GoCloak, cfg *Config) (func()) {
```

Comment:
I think that this function is not needed, it would be necessary if you want to create an event and then delete it to clear the Keycloak.